### PR TITLE
Feature/java11 neo4jv4

### PR DESCRIPTION
**Pull request by DirkMahler**

### CODE REVIEW

File: status.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Java 11
```
Comment (StephanPirnbaum) [inline status.txt]: Please remove this file

**Review by StephanPirnbaum — CHANGES_REQUESTED** [review]

File: status.txt (hunk 1 above)
Comment (DirkMahler) [reply status.txt]: Removed

**Review by StephanPirnbaum — APPROVED** [review]

File: pom.xml [hunk 2]
```diff
@@ -8,19 +8,20 @@
...
-    <artifactId>parent</artifactId>
+    <groupId>com.buschmais.jqassistant.neo4j</groupId>
+    <artifactId>embedded</artifactId>
     <version>2.0.0-SNAPSHOT</version>
-    <packaging>pom</packaging>
 
-    <name>jQAssistant Embedded Neo4j Server Parent</name>
-    <url>https://jqassistant.org/</url>
+    <name>jQAssistant Embedded Neo4j</name>
+    <url>http://jqassistant.org/</url>
```
Comment (obfischer) [inline pom.xml]: s/http/https/g

File: pom.xml [hunk 3]
```diff
@@ -168,6 +164,13 @@
                 <scope>import</scope>
                 <type>pom</type>
             </dependency>
+            <dependency>
+                <groupId>org.junit</groupId>
+                <artifactId>junit-bom</artifactId>
+                <version>${junit.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
```
Comment (obfischer) [inline pom.xml]: The JUnit BOM is already provided by the Uber parent. Shouldn't be needed here.

**Review by obfischer — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 2 above)
Comment (DirkMahler) [reply pom.xml]: Fixed.

File: pom.xml (hunk 3 above)
Comment (DirkMahler) [reply pom.xml]: Fixed

**Review by obfischer — APPROVED** [review]

**Pull request merged**